.github/workflows/test.yml

```diff
@@ -0,0 +1,51 @@
+name: Run unit and style tests
+
+on:
+  pull_request:
+  push:
+    branches:
+      - develop
+      - master
+  # Allow manually triggering the workflow.
+  workflow_dispatch:
+
+jobs:
+  test:
+    name: Run tests on PHP ${{ matrix.php }}
+    runs-on: ubuntu-latest
+    continue-on-error: ${{ matrix.experimental == true }}
+
+    strategy:
+      matrix:
+        php:
+          - '5.4'
+          - '5.5'
+          - '5.6'
+          - '7.0'
+          - '7.1'
+          - '7.2'
+          - '7.3'
+          - '7.4'
```

Comment:
What about adding PHP `8.0`, `8.1` and `8.2` ?

For PHP 8.2, you could then change the `continue-on-error: ${{ matrix.experimental == true }}` setting to: `continue-on-error: ${{ matrix.php == '8.2' }}` to allow builds against PHP 8.2 to fail while it is still under development.

Reply:
PHP 8.x will be added in next step. I want convert 1:1 state previous TravisCI state

Reply:
For 8.x is necessary add support, via https://github.com/php-parallel-lint/PHP-Var-Dump-Check/runs/5065543774?check_suite_focus=true

Reply:
That's fair enough. I'm actually tempted to suggest abandoning this repo in favour of a PHP_CodeSniffer sniff for the same at some point in the future. Should make PHP cross-version support easier ;-)